tests/ops/models/test_detection_discovery.py

```diff
@@ -190,34 +190,14 @@ def test_staged_resource_helpers(self, db: Session, create_staged_resource):
 
 
 class TestMonitorConfigModel:
-    @pytest.fixture
-    def create_monitor_config(self, db: Session, connection_config: ConnectionConfig):
```

Comment:
moved this fixture to the new `detection_discovery_fixtures.py` file I created and just renamed it to `monitor_config`, to be more consistent with our existing fixture names